public/manifest.json

```
@@ -32,7 +32,7 @@
     "notifications",
     "tabs"
   ],
-  "content_security_policy": "script-src 'self' 'wasm-eval'; object-src 'self'",
+  "content_security_policy": "default-src 'self';",
```

If we're going for strict default-src it should probably be
```suggestion
  "content_security_policy": "default-src 'none'; script-src 'self'; style-src 'self' 'unsafe-inline'; connect-src 'self' https://api.oasisscan.com https://www.oasisscan.com https://testnet.oasisscan.com https://grpc-mainnet.oasisscan.com https://grpc-testnet.oasisscan.com; frame-ancestors 'self' https: http://localhost:* http://127.0.0.1:*; img-src 'self' https: http://localhost:* http://127.0.0.1:*; base-uri 'self'; manifest-src 'self';",
```

```
default-src 'none'; 
script-src 'self'; 
style-src 'self' 'unsafe-inline'; 
connect-src 'self' https://api.oasisscan.com https://www.oasisscan.com https://testnet.oasisscan.com https://grpc-mainnet.oasisscan.com https://grpc-testnet.oasisscan.com; 
frame-ancestors 'self' https: http://localhost:* http://127.0.0.1:*; 
img-src 'self' https: http://localhost:* http://127.0.0.1:*; 
base-uri 'self'; 
manifest-src 'self';
```

---

We can reduce `img-src` if dapp repacks `siteIcon` into base64 before it sends it to us

---

@pro-wh is that part of ext-utils?

---

frame-ancestors is for ext-utils

---

Nice :slightly_smiling_face:!

Thanks for preparing this, @lukaw3d!

---

Can you also add Foundation's official public gRPC endpoints, i.e.:
```
https://grpc.oasis.dev
https://testnet.grpc.oasis.dev
```

---

Sure - do we plan to use those in config.js?

---

Yes.